Source/JavaScriptCore/inspector/agents/InspectorDebuggerAgent.cpp

```diff
@@ -1457,6 +1457,7 @@ void InspectorDebuggerAgent::didCreateNativeExecutable(JSC::NativeExecutable& na
 {
 #if ENABLE(JIT)
     auto& vm = m_debugger.vm();
+    JSC::JSLockHolder locker(vm);
```

Comment:
NIT: should this be moved lower down when we know we're for sure going to do something?  i.e. if the developer hasn't created any Symbolic Breakpoints then we will bail so acquiring the lock before that is perhaps overkill (especially since most developers likely won't have created any Symbolic Breakpoints)

Reply:
Yeah, fixed.